solr/solr-ref-guide/modules/deployment-guide/pages/backup-restore.adoc

```diff
@@ -401,6 +401,17 @@ Any children under the `<repository>` tag are passed as additional configuration
 
 Information on each of the repository implementations provided with Solr is provided below.
 
+By default, all the repository implementations verify the integrity of the index files before they are copied to the destination. However, it is possible to disable this integrity check by setting the optional configuration property `verifyChecksum`.
+
+`verifyChecksum`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: `true`
+|===
++
+Defines whether the backup repository should check the index files integrity before copying them to the destination. Set `false` to disable the checksum verification, for example to verify the integrity in a different way.
```

Comment:
[0] Is it worth calling out the encryption scenario explicitly here?  I can imagine users reading this and wondering why they would ever want to disable such a safety check...

(If we're intentionally NOT mentioning encryption because it's a 'solr-sandbox' thing, I guess I understand that as well.  Just figured I'd check)